src/ctia/stores/es/crud.clj

```diff
@@ -201,8 +237,49 @@ It returns the documents with full hits meta data including the real index in wh
   [docs ident get-in-config]
   (filter #(allow-read? % ident get-in-config) docs))
 
+(s/defschema BulkResult
+  {:errors (st/optional-keys
+            {:unauthorized [s/Str]
+             :not-found [s/Str]
+             :internal-error [s/Str]})
+   :result [{:id s/Str :action (s/enum "deleted" "updated" "indexed" "created")}]})
+
+(s/defn bulk-delete :- BulkResult
+  [{:keys [conn] :as conn-state}
+   ids :- [s/Str]
+   ident
+   es-params]
+  (let [docs-with-indices (get-docs-with-indices conn-state ids {})
+        {authorized true unauthorized false}
+        (group-by #(allow-write? (:_source %) ident)
+                  docs-with-indices)
+        missing (set/difference (set ids) (map :_id docs-with-indices))
+        prepared-docs (map #(select-keys % [:_index :_type :_id])
+                           authorized)]
+    (try
```

Comment:
Looks like the only part that can fail is `ductile.doc/bulk-delete-docs`

It tends to be better to thinly wrap in `try` just that part. That way, trivial programming errors aren't swallowed, which  makes things harder to debug